lib/omise/OmiseSearch.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+require_once dirname(__FILE__).'/res/OmiseApiResource.php';
+
+/**
+ * This class is not intended to be used directly from client code.
+ *
+ * @see OmiseCharge::search()
+ */
+class OmiseSearch extends OmiseApiResource
+{
+    const ENDPOINT = 'search';
+
+    private $dirty = true;
+    private $scope = null;
+    private $query = '';
+    private $conditions = array();
```

Comment:
@iwat ✋ how about we go with `$filters` property and `public function filter()` method name instead?
To make it conform with our document and params since those 2 were literally meant `filter` param in Omise API (unless that `$conditions` property will be used to handle something more than `filter` param).